docs/Troubleshooting.md

```diff
@@ -4,37 +4,43 @@
 ## Catalog packages or stats are missing or are not updating
 
 If you recently added the bucket or upgraded the stack, if search volume is high,
-or if the bucket is under rapid modification, wait a few minutes and try again.
+or if read/write volume is high, wait a few minutes and try again.
 
 ### Re-index the bucket
 
-1. Open the bucket in the Quilt catalog
+1. Open the bucket overview in the Quilt catalog and click the gear icon (upper right),
+or navigate to Admin settings > Buckets and inspect the settings of the bucket in question.
 
-1. Click the gear icon (upper right), or navigate to Users and buckets > Buckets
-and open the bucket in question
+1. Under "Indexing and notifications", click "Re-index and Repair".
 
-    ![](imgs/admin-bucket.png)
+> Optionally: **if and only if** bucket notifications are not working and you are
+> certain that there are no other subscribers to the S3 Events of the bucket in
+> question, check "Repair S3 notifications".
 
-1. Under "Indexing and notifications", click "Re-index and Repair". Optional:
-if and only if bucket notifications were deleted or are not working,
-check "Repair S3 notifications".
+Bucket packages, stats, and the search index will repopulate in the next few minutes.
+Buckets with more than one million objects will take longer.
 
-1. Wait a few minutes while bucket statistics and packages repopulate
+### Inspect the Elasticsearch domain
 
-### Inspect Elasticsearch domain
+1. Determine your Quilt instance's ElasticSearch domain from Amazon Console > OpenSearch
+or `aws opensearch list-domain-names`. Note the domain name (hereafter `QUILT_DOMAIN`).
```

Comment:
Maybe get it from cloudformation by its logical ID?

Reply:
I decided not to even mention this because e.g. in the new TF installs it's not a CF resource at all.